CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
-- Upgrade express de from 4.19.2 to 4.20.0
-- Upgrade mongoose dep from 5.13.20 to 5.13.22
+- Upgrade express dep from 4.19.2 to 4.20.0
+- Upgrade mongodb devdep from 4.17.1 to 4.17.2
+- Upgrade mongoose dep from 5.13.22 to 8.4.4 (solving vulnerability CVE-2024-53900) (#1674)
```

Comment:
```suggestion
- Upgrade mongoose dep from 5.13.20 to 8.4.4 (solving vulnerability CVE-2024-53900) (#1674)
```

By the existing CNR, I undersand previous version uses 5.13.20, so the line should be fixed that way.